### DISCUSSION TITLE
How can I go further?

### DISCUSSION THREAD

**Discussion by Luismi1984** [neonbjb/tortoise-tts#693] [Q&A]
Hello, I am Luismi and I completely new here.
I mean new in almost everything: "Python", "Git"...

I followed these steps:

`conda create --name tortoise python=3.9 numba inflect
conda activate tortoise
conda install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch -c nvidia
conda install transformers=4.29.2
git clone https://github.com/neonbjb/tortoise-tts.git
cd tortoise-tts
python setup.py install`

But I do not know how can I go further.
Can someone help me here, please?
Thank you in advance!

**Comment by Luismi1984** [comment]
My excuses: I forgot to mention that I use Windows

**Reply by Chapoly1305** [reply]
what's your graphic card?

**Reply by Luismi1984** [reply]
NVIDIA, GeForce RTX 3060
